website_legal_page/views/website_privacy.xml

```diff
@@ -6,17 +6,17 @@
           inherit_id="website.layout_footer_copyright"
           name="Privacy policy Link">
     <xpath expr="//footer//div/span[@t-field='res_company.name']" position="after">
-        <span>- <a href="/page/privacy">Privacy policy</a></span>
+        <span>- <a href="/legal/privacy">Privacy Policy</a></span>
```

Comment:
It turns out that it is better for SEO having a explicit name than a short one (unless the explicit one is too long). What if we take the chance to rename this one to `/legal/privacy-policy`?